app/tests/unit/pages/login-redirect.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import LoginRedirect from 'pages';
+import {loginRedirectQueryResponse} from 'loginRedirectQuery.graphql';
+
+const query: loginRedirectQueryResponse['query'] = {
+  session: null,
+  openedReportingYear: {
+    applicationCloseTime: '2020-08-31 23:59:59.999-07'
+  },
+  nextReportingYear: {
+    applicationCloseTime: '2021-12-30 14:49:54.191-08'
+  }
+};
+
+describe('Login redirect page', () => {
+  it('It matches the last accepted Snapshot', () => {
+    const wrapper = shallow(<LoginRedirect query={query} />);
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('It passes an application deadline to the RegistrationLoginButtons component', () => {
+    const wrapper = shallow(<LoginRedirect query={query} />);
+    expect(
+      wrapper
+        .find('RegistrationLoginButtons')
+        .first()
+        .prop('applicationDeadline')
+    ).toBe('August 31, 2020');
+  });
+});
```

Comment:
In the course of this copy (date) update, I realized there was no test for this page. Added one here, as a test would have alerted me earlier to a failing build (I'd been unaware the RegistrationLoginButtons component I updated for the landing page is also present on this page).